hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -341,6 +341,17 @@ profile::jenkinscontroller::jcasc:
               - linux
             useAsMuchAsPosible: true
             spot: true
+            userData:
```

Comment:
```suggestion
            userData: &datadogInitScript
```
Use of YALM anchores and aliases: https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet

Reply:
Careful: YAML anchor does not allow to merge array (https://github.com/yaml/yaml/issues/35)

Reply:
Merge not used here?

Reply:
nope but the question was raised by Stephane since both Initscript (azure Linux VM templates) and userData (EC2 Linux templates) have the same set of shell instructions